examples/quickstart/jupyter-notebooks/druidapi/README.md

```diff
@@ -36,21 +39,25 @@ At present, the best way to use `druidapi` is to clone the Druid repo itself:
 git clone git@github.com:apache/druid.git
 ```
 
-`druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`
+`druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`.
+From this directory, install the package and its dependencies with pip using the following command:
```

Comment:
Took me a moment to realize you had introduced another directory level. That is helpful: there is not a place to put the test code that I currently have sitting off to the side. Would be good to mention this structure to make it obvious that there there are two `druidapi` directories here.

Reply:
Updated